website/docs/docs/build/exposures.md

```diff
@@ -24,6 +24,39 @@ Exposures make it possible to define and describe a downstream use of your dbt p
 
 Exposures are defined in `.yml` files nested under an `exposures:` key.
 
+<VersionBlock firstVersion="1.4">
+
+<File name='models/<filename>.yml'>
+
+```yaml
+version: 2
+
+exposures:
+
+  - name: weekly_jaffle_metrics
+    type: dashboard
+    maturity: high
+    url: https://bi.tool/dashboards/1
+    description: >
+      Did someone say "exponential growth"?
+
+    depends_on:
+      - ref('fct_orders')
+      - ref('dim_customers')
+      - source('gsheets', 'goals')
+      - metric('example_metric_name')
+
+    owner:
+      name: Claire from Data
```

Comment:
At the risk of depriving Claire of some ongoing notoriety, we might swap this out for one of our newer cast of characters:
```suggestion
      name: Callum McData
```